config/sso.go

```diff
@@ -21,6 +21,7 @@ type SSOConfig struct {
 	CustomGroupClaimName string `json:"customGroupClaimName,omitempty"`
 	UserInfoPath         string `json:"userInfoPath,omitempty"`
 	InsecureSkipVerify   bool   `json:"insecureSkipVerify,omitempty"`
+	FilterSSOGroupsRegex string `json:"filterSSOGroupsRegex,omitempty"`
```

Comment:
```suggestion
	FilterGroupsRegex string `json:"filterGroupsRegex,omitempty"`
```

"SSO" is redundant here as this is a field of `SSOConfig`

Reply:
done